python/cuml/tests/test_hdbscan.py

```diff
@@ -475,6 +475,25 @@ def test_hdbscan_core_dists_bug_4054():
     assert adjusted_rand_score(cu_labels_, sk_labels_) > 0.99
 
 
+@pytest.mark.parametrize(
+    "_metric,expected_to_fail",
```

Comment:
I know I used that name in the example, but let's rename `expected_to_fail` to `supported` and switch the logic.
Further, please rename the `_metric` parameter to `metric`.